tests/tests_ntp.yml

```diff
@@ -1,6 +1,9 @@
 - name: Configure time synchronization with NTP servers
   hosts: all
   vars:
+    timesync_chrony_custom_settings:
```

Comment:
will need to document this parameter in README.md
will need some sort of test to verify the settings have been applied